src/store/ui/reducers.ts

```diff
@@ -106,6 +107,12 @@ const uiReducer = (
         },
       };
     }
+    case UI_DISCONNECT_ALL_ANCHORS: {
```

Comment:
Is this reducer actually more like reset sidenote state?

Just looking at it it does more than the anchors, also removes all comments across all docs?

Reply:
yes is takes out the full state